README.md

```diff
@@ -86,10 +86,12 @@ Let's look at different ways to create a Hash.
 => {}
 > apartment[:address]
 => ""
-> apartment[:address] = { street: '255 Long Road', city: 'Awesomeville', bedrooms: 3}
-=> {:street=>"255 Long Road", :city=>"Awesomeville", :bedrooms=>3}
-> apartment.merge({rent: 1000})
-=> {:street=>"255 Long Road", :city=>"Awesomeville", :bedrooms=>3, :rent=>1000}
+> apartment[:address] = { street: '255 Long Road', city: 'Awesomeville'}
```

Comment:
Style error. I know it isn't obvious because it's in a README instead of something linted by rubocop. Check your spacing inside hashes.